Library/src/Systems/Conversation/ConversationSystemInternal.cpp

```diff
@@ -46,6 +46,25 @@ namespace
 
         return true;
     }
+
+    bool EnsureUserHasPermission(const common::String& UserId, const common::String& ConversationUserId, bool IsConversation)
+    {
+        if (UserId != ConversationUserId)
+        {
+            if (IsConversation)
+            {
+                CSP_LOG_ERROR_MSG("User does not have permission to modify this conversation.");
+            }
+            else
+            {
+                CSP_LOG_ERROR_MSG("User does not have permission to modify this message.");
+            }
```

Comment:
[Comment] This is the sort of stuff I wish we had `libfmt` for
```suggestion
           CSP_LOG_ERROR_MSG(libfmt::fmt("User does not have permission to modify this {}.", IsConversation ? "conversation" : "message"));
```

Doable without it, but more clumsy and often creates unnecessary copies.

Reply:
This would be a nice to have